mac/installation.md

```diff
@@ -88,7 +88,7 @@ To install Visual Studio for Mac behind a firewall, certain endpoints must be ma
 
 Configure your network to allow access to the following locations:
 
-* [Visual Studio endpoints](/visualstudio/install/install-visual-studio-behind-a-firewall-or-proxy-server)
+* [Visual Studio endpoints](../docs/install/install-and-use-visual-studio-behind-a-firewall-or-proxy-server)
```

Comment:
This one i'm not 💯 on because it is partially local, so this will be better for GH, but I'm not sure how openpublishing will handle this since they seem to be in separate collections (docs vs. mac)